test/testdata/fixtures/add-dc/k8ssandra.yaml

```diff
@@ -29,28 +29,30 @@ spec:
         concurrent_reads: 2
         concurrent_writes: 2
         concurrent_counter_writes: 2
+        cas_contention_timeout_in_ms: 60000
+        counter_write_request_timeout_in_ms: 60000
+        range_request_timeout_in_ms: 60000
+        read_request_timeout_in_ms: 60000
+        request_timeout_in_ms: 60000
+        truncate_request_timeout_in_ms: 60000
+        write_request_timeout_in_ms: 60000
       jvmOptions:
         heapSize: 384Mi
     networking:
       hostNetwork: true
-    resources:
```

Comment:
If there's no resources limits at all, then this Cassandra could eat everything. It shouldn't, why isn't 1GB enough in these tests?

Reply:
I think it's not really a concern for e2e tests. If Cassandra eats more than it should and gets killed, the test fails. Same if it eats everything and doesn't get killed, it fails.
Limits were incorrectly set in many tests and I don't think they provide value for e2e tests anyway.
Also, there's no way Cassandra can eat more than twice the heap size, plus MCAC and MAAC heap sizes*2. So there's an absolute limit, it's just not set at the k8s level.

Reply:
If it can't eat, then there should be nothing wrong with setting it a limit, right? That limit is never reached.

Reply:
What's wrong with setting a limit is if it's incorrectly set, which was the case for various test scenarios.
Setting the limit correctly is error prone in this case because of the various settings that can influence Cassandra's pod memory usage (Cassandra heap+off heap, Management API heap+off heap, MCAC collectd), and if heap sizes are changed, the limits must be changed accordingly.
Since this work aims at improving the test pass rate, we're better off without limits.

Anyone can work on a follow up ticket to appropriately set those limits and run tens of executions to verify they are set appropriately, but IMO removing them altogether serves the primary goal here.